backend/de.metas.business/src/main/java/de/metas/product/ProductRepository.java

```diff
@@ -382,6 +382,18 @@ private Product ofProductRecord(@NonNull final I_M_Product productRecord)
 				.ean(productRecord.getUPC())
 				.orgId(OrgId.ofRepoId(productRecord.getAD_Org_ID()))
 				.procurementStatus(productRecord.getProcurementStatus())
+				.selfPacked(productRecord.isSelfPacked())
+				.dimensionsInCM(getProductDimensions(productRecord))
+				.build();
+	}
+
+	@NonNull
+	private DimensionsInCM getProductDimensions(final @NonNull I_M_Product productRecord)
```

Comment:
static

Reply:
name it extract(Product)Dimensions  to be consistent with other names

Reply:
done